howfairis/workarounds/github_caching.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+from datetime import timedelta
+import requests
+from dateutil import tz
+
+
+def github_caching_check(checker):
```

Comment:
We can use GitHub API to get info about the commits. https://docs.github.com/en/rest/reference/repos#commits

we can use `path` parameter to point to README.md or README.rst
and then get timestamp info

Reply:
In addition to what Faruk suggests, I also tested with `?page=0&per_page=1`, so you get an array of 1 element back

Reply:
Ok, both suggestions have been implemented in `howfairis/workarounds/github_caching.py`